PdCore/src/org/puredata/android/service/PdService.java

```diff
@@ -245,12 +245,18 @@ private void invokeSetForeground(boolean foreground) {
 		protected static final int NOTIFICATION_ID = 1;
 		private boolean hasForeground = false;
 
-		@SuppressWarnings("deprecation")
 		protected Notification makeNotification(Intent intent, int icon, String title, String description) {
 			PendingIntent pi = PendingIntent.getActivity(getApplicationContext(), 0, intent, 0);
-			Notification notification = new Notification(icon, title, System.currentTimeMillis());
-			notification.setLatestEventInfo(PdService.this, title, description, pi);
-			notification.flags |= Notification.FLAG_ONGOING_EVENT;
+			Notification notification = new Notification.Builder(PdService.this)
+					.setSmallIcon(icon)
```

Comment:
Notification.Builder was added in API 11, whereas the minSdkVersion for PdCore is currently 11. Bump the minSdkVersion to 11?